sherpa/bin/streaming_pruned_transducer_statelessX/streaming_server.py

```diff
@@ -76,6 +77,7 @@ def get_args():
     parser.add_argument(
         "--bpe-model-filename",
         type=str,
+        required=True,
```

Comment:
This one causes the following CI to fail:
https://github.com/k2-fsa/sherpa/runs/7474188404?check_suite_focus=true

```
streaming_server.py: error: the following arguments are required: --bpe-model-filename
```

For Chinese ASR, it uses Chinese characters as modeling units and does not use BPE models. In this case, `--bpe-model-filename` can be omitted since we can provide `--token-filename`.